packages/visualizations/src/components/Map/utils.ts

```diff
@@ -268,16 +268,12 @@ export const computeTooltip: ComputeTooltipFunction = (
         }
     }
 
-    const tooltipRawValues: {
-        value?: number;
-        label: string;
-        key: string;
-    } = {
-        value: matchedFeature?.y,
+    const tooltipRawValues: TooltipParams = {
+        value: matchedFeature?.y.toString(),
```

Comment:
It's tied to changes on the Studio side, but I'm not sure why here, where we know we start from a number, we have to transform it into a string, then pass that string to `Intl.NumberFormat` which expects a number. Doing number -> string -> number in the middle is a risk of losing information (for example truncating values in very long floats), and I'm not sure what the gain is. Also it seems logical to expect a value that is a number here, in terms of function signature / API, because in the context of a choropleth it will always be a number underneath.

Reply:
Indeed it is a very good remark, thank you. I must admit I didn't think much about it when I decided to change the type of the `TooltipParams` interface. Fixed [in this branch](https://github.com/opendatasoft/ods-dataviz-sdk/pull/131/commits/6d6f829ca3a06d566ba75303d090996e7d94f941) and in the [Studio as well](https://github.com/opendatasoft/platform/pull/9903/commits/f78847a8d16ca5604a116f7d17982eed7a1ae190).